tests/lean/interactive/rwElabConst.lean

```diff
@@ -0,0 +1,12 @@
+import Lean
+open Lean PrettyPrinter Delaborator SubExpr
+@[delab app.Eq]
+def checkUniv : Delab := do
+  let .const _ [.mvar u] := (← getExpr).getAppFn | failure
+  _ ← u.getLevel
+  failure
+
```

Comment:
For those not deep in the material, please include a comment what this `delab` does and why it’s needed to test what this is testing?